app/blueprints/assessments/templates/macros/fund_dashboard_summary.html

```diff
@@ -57,6 +74,8 @@ <h3 class="govuk-body govuk_summary-card__title govuk-!-font-size-24">{{round_na
                     View all submitted applications
                 </a>
             {% endif %}
+            {{ round_links(access_controller, export_href, assessment_tracker_href) }}
+
```

Comment:
these are assessment related links, it makes sense to show these after assessment round is open? (Does the link work when application is live ?)

Reply:
Hmm, I think (from Sudheer) we are moving towards asynchronous assessment support, whilst the round is live.  I’ll double check these links actually work!

Reply:
```suggestion
            {{ round_links(access_controller, export_href, assessment_tracker_href) }}
            {% endif %}            
```

we can move this to above if condition block for now. Can be viewed by setting the env variable to True. Later can decide what links to show on Prod

Reply:
Which env var sorry?

Reply:
`SHOW_ASSESSMENTS_LIVE_ROUNDS`

Reply:
Thanks

Reply:
Hi Navya - sorry for delay, have added this in https://github.com/communitiesuk/funding-service-design-assessment/pull/413/commits/41a5bc7020907466c9837002c8480d2ea80f9917

I've also refactored the template to just pass in summary vs unpacking the dict version